src/status_im/ui/screens/chat/message/link_preview.cljs

```diff
@@ -88,16 +98,57 @@
                           :style styles/link-preview-site}
                 site]])]])))))
 
+(defview community-preview [community outgoing timeline]
+  (let [{:keys [name members description verified]} community
+        members-count (count members)]
+    [react/view (styles/link-preview-wrapper outgoing timeline)
+     (if verified [quo/text {:size :small
+                             :color :link
+                             :style styles/community-preview-header}
+                   (i18n/label :t/verified-community)]
+         [quo/text {:size :small
+                    :color :secondary
+                    :style styles/community-preview-header}
+          (i18n/label :t/community)])
+     [quo/separator]
+     [react/view {:flex-direction :row :align-self :flex-start :margin 12}
+      [communities.icon/community-icon community]
+      [react/view {:flex 1 :flex-direction :column :margin-left 12}
+       [quo/text {:weight :bold :size :large} name]
+       [quo/text description]
+       [quo/text {:size :small
+                  :color :secondary}
+        (i18n/label-pluralize members-count :t/community-members {:count members-count})]]]
+     [quo/separator]
+     [quo/button {:on-press #(re-frame/dispatch [:navigate-to
+                                                 :community
+                                                 {:community-id (:id community)}])
+                  :type     :secondary}
+      (i18n/label :t/view)]]))
+
+(defview community-preview-loader [community-link outgoing timeline]
+  (letsubs [cache [:link-preview/cache]]
+    (let [community (get cache community-link)
+          community-id (community-id-from-link community-link)]
+      (if (not community)
+        (do
+          (re-frame/dispatch
```

Comment:
You will want to run this in a `component-did-mount`, otherwise it will run at each render, but there might be better options
@flexsurfer or @shivekkhurana will be able to suggest what's best